### ISSUE TITLE
Secondary addon module issue

### ISSUE THREAD

**Issue by vaDrn** [chan-sccp/sccp_manager#25]
SEPXXX.cnf.xml file generated by `sccp manager` contains only one entry for an addon module. 
I tried adding cisco 7960 with two 7914 expansion modules and cisco 7965 with two 7916 modules with same results. Generated .cnf.xml file looks like this: 
```
<addOnModules><addOnModule idx="1"><loadInformation>S00105000400</loadInformation></addOnModule></addOnModules></device>
```

Everything works without issues if I add second addon module entry manually through text editor:
```
  <addOnModules>
   <addOnModule idx="1"><loadInformation>S00105000400</loadInformation></addOnModule>
   <addOnModule idx="2"><loadInformation>S00105000400</loadInformation></addOnModule>
  </addOnModules>
</device>

```

**Comment by dkgroot** [comment]
Hmm I thought i had already addressed this in one but apparently was only a partial PR
https://github.com/PhantomVl/sccp_manager/pull/105

@steve-lad Should at least give an indication where to look (I hope)

**Comment by steve-lad** [comment]
@dkgroot Could you please copy/move this PR to chan-sccp/sccp_manager?

**Comment by steve-lad** [comment]
@vaDrn Please could you verify this fix - the xml in the cnf is not so elegant but should work. 
@dkgroot has proposed a separate modification to tidy up the xml

**Comment by steve-lad** [comment]
Fixes issue PhantomVl/sccp_manager#104

**Comment by vaDrn** [comment]
Yes, everything seems to be working fine. Generated cnf file looks correct. @steve-lad HUGE thanks for you work
